### ISSUE TITLE
Downloading Forge Installer stuck at 0 % when installing a new pack

### ISSUE THREAD

**Issue by SuperWario128** [killpowa/Twickt-Launcher#1091]
**Describe the bug**
When installing a new Forge instance, when it gets to Downloading Forge Installer, it stays at 0%.
**To Reproduce**
1. Go to '...'Add New Instance
2. Click on '....'Vanilla > Forge
3. Scroll down to '....'1.16.5 > 36.2.0
Downloading Forge Installer... stuck at 0% and doesn't progress.
**Expected behavior**
Forge is supposed to install, but doesn't.
**Possible solutions**
Some ideas of how you would solve this issue, if possible

**Screenshots**

**Operating System:**
- OS: [e.g. Windows] Windows
- Version: [e.g. 10] 10
- Java Version: [e.g. 8/16] Current
- Minecraft Version: [e.g. 1.16.5] 1.16.5
- Mod Loader: [e.g. Forge/Fabric] Forge

**Additional context**

**Comment by joshbenner** [comment]
Console shows:

```
asyncToGenerator.js:6 Uncaught (in promise) Error: certificate has expired
    at TLSSocket.onConnectSecure (_tls_wrap.js:1497)
    at TLSSocket.emit (events.js:315)
    at TLSSocket._finishInit (_tls_wrap.js:932)
    at TLSWrap.ssl.onhandshakedone (_tls_wrap.js:706)
```

**Comment by huantianad** [comment]
Duplicate of #1086

**Comment by Eskaan** [comment]
Fixed in latest v1.1.15-beta.5 (latest beta release)
